docs/reference/management_commands.md

```diff
@@ -6,11 +6,6 @@
 
 ## publish_scheduled
 
-```{versionchanged} 4.1
```

Comment:
Are we keeping this alias indefinitely, or should we start a deprecation process?